src/Runtime/jni/jnilog.c

```diff
@@ -140,21 +214,25 @@ static FILE *get_log_file_by_name(char *name) {
  */
 void log_init() {
 
-  if (log_inited)
+#ifdef USE_PTHREAD_SPECIFIC
```

Comment:
Why not add another helper function to encapsulate this? Then log_init won't have any ifdefs and will just call helper functions.

Reply:
Was lazy to deal with it since it's just one line. But it's good to get rid of the #ifdef so I used a PTHREAD_ONCE macro. Thanks.